[BUG] report: cog throws error

```py
[2023-05-10 00:57:23] [ERROR] red: Exception in command 'report'
Traceback (most recent call last):
  File "/home/redbot/.local/lib/python3.11/site-packages/discord/ext/commands/core.py", line 229, in wrapped
    ret = await coro(*args, **kwargs)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/redbot/data/cogs/CogManager/cogs/report/report.py", line 84, in cmd_report
    data = self.make_report_embed(ctx, message, emergency=False)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/redbot/data/cogs/CogManager/cogs/report/report.py", line 180, in make_report_embed
    .set_author(name="Report", icon_url=ctx.author.avatar_url)
                                        ^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'Member' object has no attribute 'avatar_url'
```

--- a/enforcer/enforcer.py
+++ b/enforcer/enforcer.py
@@ -70,7 +70,7 @@ class EnforcerCog(commands.Cog):
         author = message.author
 
         data = discord.Embed(color=discord.Color.orange(), description=message.content)
-        data.set_author(name=f"Message Enforced - {author}", icon_url=author.avatar_url)
+        data.set_author(name=f"Message Enforced - {author}", icon_url=author.display_avatar.url)
         data.add_field(name="Enforced Reason", value=reason, inline=True)
         data.add_field(name="Channel", value=message.channel.mention, inline=True)
 

--- a/report/report.py
+++ b/report/report.py
@@ -177,7 +177,7 @@ class ReportCog(commands.Cog):
                 colour=discord.Colour.red() if emergency else discord.Colour.orange(),
                 description=escape(message or "<no message>"),
             )
-            .set_author(name="Report", icon_url=ctx.author.avatar_url)
+            .set_author(name="Report", icon_url=ctx.author.display_avatar.url)
             .add_field(name="Reporter", value=ctx.author.mention)
             .add_field(name="Channel", value=ctx.channel.mention)
             .add_field(name="Timestamp", value=f"<t:{int(ctx.message.created_at.timestamp())}:F>")

--- a/report/report.py
+++ b/report/report.py
@@ -190,7 +190,7 @@ class ReportCog(commands.Cog):
                 colour=discord.Colour.red() if emergency else discord.Colour.orange(),
                 description=escape(message or "<no message>"),
             )
-            .set_author(name="Report Received", icon_url=ctx.author.avatar_url)
+            .set_author(name="Report Received", icon_url=ctx.author.display_avatar.url)
             .add_field(name="Server", value=ctx.guild.name)
             .add_field(name="Channel", value=ctx.channel.mention)
             .add_field(name="Timestamp", value=f"<t:{int(ctx.message.created_at.timestamp())}:F>")

--- a/timeout/timeout.py
+++ b/timeout/timeout.py
@@ -47,10 +47,10 @@ class Timeout(commands.Cog):
             inline=False
         )
 
-        if user.avatar_url:
+        if user.display_avatar:
             embed.set_author(
                 name=f"{user} {action_info['action']} timeout",
-                icon_url=user.avatar_url)
+                icon_url=user.display_avatar.url)
         else:
             embed.set_author(
                 name=f"{user} {action_info['action']} timeout"
@@ -236,7 +236,7 @@ class Timeout(commands.Cog):
         )
         embed.set_author(
             name="Timeout Cog Settings",
-            icon_url=ctx.guild.me.avatar_url
+            icon_url=ctx.guild.me.display_avatar.url
         )
         embed.add_field(
             name="Log Channel",

--- a/verify/verify.py
+++ b/verify/verify.py
@@ -419,7 +419,7 @@ class VerifyCog(commands.Cog):
         if log_id:
             log = guild.get_channel(log_id)
             data = discord.Embed(color=discord.Color.orange())
-            data.set_author(name=f"{message} - {user}", icon_url=user.avatar_url)
+            data.set_author(name=f"{message} - {user}", icon_url=user.display_avatar.url)
             data.add_field(name="User", value=user.mention)
             data.add_field(name="ID", value=user.id)
             if not failmessage:

